docs/index.md

```diff
@@ -28,3 +28,7 @@ provider "sym" {
 ### Required
 
 - `org` (String) Your Sym Org ID
+
+### Optional
+
+- `jwt_env_var` (String) Environment variable storing your Sym Access Key
```

Comment:
While access key is Not Wrong, we've been referring to these types of tokens as Bot Tokens (don't we love unconventional jargon?), so I would probably refer to it as a Bot Token here as well.

```suggestion
- `jwt_env_var` (String) Environment variable storing your Sym Bot Token
```

I'd also recommend updating the `Authentication` section above to indicate that the new order of authentication is:

1. The value stored in the `jwt_env_var`environment variable passed into the `sym` provider block
2. The `SYM_JWT` environment variable
3. The `symflow login` config